doc/lua_api.txt

```diff
@@ -4375,6 +4375,14 @@ Call these functions only at load time!
 * `minetest.register_on_leaveplayer(function(ObjectRef, timed_out))`
     * Called when a player leaves the game
     * `timed_out`: True for timeout, false for other reasons.
+* `minetest.register_player_on_step(function(ObjectRef, dtime))`
+    * Called for every player on every server step.
+    * `ObjectRef`: The player.
+    * `dtime`: Time from last step (see also `minetest.register_globalstep`).
+* `minetest.register_player_on_slow_step(step_size, function(ObjectRef))`
```

Comment:
I think a better name for this would be:

```suggestion
* `minetest.register_player_on_interval(step_size, function(ObjectRef))`
```

Reply:
Done.